postgres_to_es/loader.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List
+import requests
+import json
+from urllib.parse import urljoin
+from models import AbstractLoader
+from requests.exceptions import Timeout
+logger = logging.getLogger()
+
+
+class ESLoader(AbstractLoader):
+    def __init__(self, url: str):
+        self.url = url
+
+    def load(self, data):
+        self.load_to_es(data, 'movies')
+
+    def _get_es_bulk_query(self, rows: List[dict], index_name: str) -> List[str]:
+        """
+        Подготавливает bulk-запрос в Elasticsearch
+        """
+        prepared_query = []
+        for row in rows:
+            prepared_query.extend([
+                json.dumps({'index': {'_index': index_name, '_id': row['id']}}),
+                json.dumps(row)
+            ])
+        return prepared_query
+
+    def load_to_es(self, records: List[dict], index_name: str):
+        """
+        Отправка запроса в ES и разбор ошибок сохранения данных
+        """
+        prepared_query = self._get_es_bulk_query(records, index_name)
+        str_query = '\n'.join(prepared_query) + '\n'
+
+        response = requests.post(
+            urljoin(self.url, '_bulk'),
+            data=str_query,
+            headers={'Content-Type': 'application/x-ndjson'},
+            timeout=5
+        )
+
+        json_response = json.loads(response.content.decode())
```

Comment:
У класса `Response` есть встроенный метод `.json()`, можно сразу написать:
```suggestion
        json_response = response.json()
```

Ещё здесь не проверяется код возврата (response.status_code или response.raise_for_status())